sdk/servicebus/service-bus/README.md

```diff
@@ -85,90 +71,100 @@ The following sections provide code snippets that cover some of the common tasks
 
 ### Send messages
 
-Once you have created an instance of a `QueueClient` or `SubscriptionClient` class, create a sender
-using the [createSender](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/queueclient#createsender--)
-function. This gives you a sender which you can use to [send](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/sender#send-sendablemessageinfo-) messages.
+Once you have created an instance of a `ServiceBusClient` class, you can get a `Sender`
+using the [getSender](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#getsender-string-)
+function. This gives you a sender which you can use to [send](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/sender#send-servicebusmessage-)
```

Comment:
Use the term `method` when the function is on a class. Use `function` when it is a standalone function

Reply:
```suggestion
method. This gives you a sender which you can use to [send](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/sender#send-servicebusmessage-)
```